linkerd/system/Cargo.toml

```diff
@@ -14,4 +14,4 @@ tracing = "0.1"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 libc = "0.2"
-procinfo = "0.4"
+procfs = "0.15.1"
```

Comment:
should probably end with a newline here afaik that's what POSIX standards define as a line, no need to remove it.

Reply:
Weird, running `just fmt` removed the line. I will add it back